.github/drafts/16506/16506_category_slider_outline.md

```diff
@@ -0,0 +1,62 @@
+*** Category Slider ***
+
+The Category Slider widget allows users to choose from a fixed set of incremental values.
+
+< TODO: NEED VIDEO DEMO >
+
+---
+# Usage
```

Comment:
Usage can be a part of the slider landing page where you could explain the sample use cases for each slider.